styx-cli/src/main/java/com/spotify/styx/cli/CliMain.java

```diff
@@ -694,6 +698,10 @@ private void retryWorkflowInstance() throws ExecutionException, InterruptedExcep
         backfillCreate.addArgument("-d", "--description")
             .help("a description of the backfill");
     final Argument backfillCreateEnv = addEnvVarArgument(backfillCreate, "-e", "--env");
+    final Argument backfillCreateAllowFuture = addEnvVarArgument(backfillCreate, "--allow-future")
```

Comment:
Usual name for this kind of flag that disables safeguards is `--force`

Reply:
Yeah, thought about this. But it seems too generic for this use case.

Reply:
So the case could be if the instance is already active, `--force` might just silence the error.